packages/ceramic-core/src/ceramic.ts

```diff
@@ -88,6 +93,20 @@ class Ceramic {
     return this._docmap[id]
   }
 
+  async pinDocument (docId: string): Promise<void> {
+    const document = await this.loadDocument(docId)
+    await this.dispatcher.pinningService.pin(document)
+  }
+
+  async unpinDocument (docId: string): Promise<void> {
+    const document = await this.loadDocument(docId)
+    await this.dispatcher.pinningService.rm(document)
+  }
+
+  async listPinned (docId?: string): Promise<any> {
+    return this.dispatcher.pinningService.ls(docId)
+  }
```

Comment:
Looks like api now is:
* `ceramic.pinDocment`
* `ceramic.unpinDocment`
* `ceramic.listPinned`

We want to mirror the [ipfs api](https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/PIN.md), so it should look like this:

* `ceramic.pin.add(docId)`
* `ceramic.pin.rm(docId)`
* `ceramic.pin.ls(docId?)`